.changeset/friendly-clocks-act.md

```diff
@@ -0,0 +1,13 @@
+---
+'create-astro': patch
+'@astrojs/telemetry': patch
```

Comment:
I doesn't seem like the telemetry package is changed, I think we can remove this?

Reply:
Okay.